client/allocrunner/taskrunner/task_runner_hooks.go

```diff
@@ -105,6 +105,18 @@ func (tr *TaskRunner) initHooks() {
 			logger:    hookLogger,
 		}))
 	}
+
+	// If there are any script checks, add the hook
+	scriptCheckHook := newScriptCheckHook(scriptCheckHookConfig{
+		alloc:  tr.Alloc(),
+		task:   tr.Task(),
+		consul: tr.consulClient,
+		logger: hookLogger,
+	})
+	if scriptCheckHook != nil {
```

Comment:
I don't seen a place in `newScriptCheckHook` where nil could be returned. Do we need this check?

Reply:
I fixed that for https://github.com/hashicorp/nomad/pull/6197/files/0c867af9f98c9a7b645f3815f0864989d3c52e69#diff-b6fc0c91a5c6cce3a0c98410d10f46e5 but missed removing it here. Thanks, will fix.